data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -369,12 +301,11 @@ else if(nodeSelExpr.getChildCount()==2 || nodeSelExpr.getChildCount()==1 && node
 	}
 
 	private void handleTokAllColRef(ASTNode nodeSelExpr) {
-		System.out.println("Input column = *");;
-//		columnList.add("input->*");
+		LOGGER.info("Input column = *");;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)